test/utils.js

```diff
@@ -22,3 +22,19 @@ exports.getTestPages = function() {
     };
   });
 };
+
+exports.prettyPrint = function(html) {
+  return prettyPrint(html, {
+    "indent_size": 4,
+    "indent_char": " ",
+    "indent_level": 0,
+    "indent_with_tabs": false,
+    "preserve_newlines": false,
+    "break_chained_methods": false,
+    "eval_code": false,
+    "unescape_strings": false,
+    "wrap_line_length": 0,
```

Comment:
I would really like lines to wrap, actually... this diff is hard to review in part because of the lack of wrapping. Can we change this?